### ISSUE TITLE
Mass Obsoletion: focus on branch 'respiratory system disorder' (MONDO:0005087)

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6754]
branch:  'respiratory system disorder' (MONDO:0005087)

To be obsoleted: 

obsoletion candidate ID | obsoletion candidate label
-- | --
http://purl.obolibrary.org/obo/MONDO_0017040 | exposure-related interstitial lung disease
http://purl.obolibrary.org/obo/MONDO_0017030 | interstitial lung disease in childhood and adulthood
http://purl.obolibrary.org/obo/MONDO_0017036 | Langerhans cell histiocytosis in childhood and adulthood
http://purl.obolibrary.org/obo/MONDO_0020537 | occupational allergic alveolitis
http://purl.obolibrary.org/obo/MONDO_0017031 | primary interstitial lung disease in childhood and adulthood
http://purl.obolibrary.org/obo/MONDO_0017027 | primary interstitial lung disease specific to adulthood
http://purl.obolibrary.org/obo/MONDO_0015930 | respiratory malformation
http://purl.obolibrary.org/obo/MONDO_0017034 | secondary interstitial lung disease in childhood and adulthood

**Comment by twhetzel** [comment]
@nicolevasilevsky the spreadsheet is ready for review for both orphaned and leave the branch terms.

**Comment by nicolevasilevsky** [comment]
Thanks @twhetzel!

**Comment by nicolevasilevsky** [comment]
looks good! This spreadsheet is ready for PR now

**Comment by sabrinatoro** [comment]
Expert review feedback: https://docs.google.com/spreadsheets/d/1yJ08nxP3Eu59G3wwVO8KXc6gw6Mh21D2/edit#gid=547601471 


TO DO:
- [ ] add new parents ("ROBOT Reviewer-Curator Agree - Stay" Tab)
- [ ] add new comments ("ROBOT Reviewer-Curator Agree - Leave" Tab) !!!!1 term comment will need to be added manually
- [ ] add new parents ("ROBOT Reviewer-Stay Curator-Leave" Tab)


Note: Once the to-do list is done, all terms will have been reviewed and this issue can be closed

**Comment by sabrinatoro** [comment]
There are some comments from the community that should be reviewed, but this is future work. This issue can be closed for now.
